### ISSUE TITLE
Quest: Supplies for the Crossroads

### ISSUE THREAD

**Issue by Kalimor** [dalaranwow/dalaran-wow#5359]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who) I am in the barrens and trying to do the Supplies for the crossroads quest. But when i try to pick up the crates for the quest. It shows a cast bar when picking the item up but when it finishes nothing happens. And after that when i try again, it says ''That object is already being used''.

**Expected behaviour**: Tell us what should happen instead. I should get the item that is required for the quest and then being able to turn in the quest. 

**Steps to reproduce the problem**:

1. Accept the quest ''Supplies for the Crossroads.
2. Try to pick up the supplies. 
3. Fail, because you cant pick them up.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
